tokio-tls/examples/echo.rs

```diff
@@ -0,0 +1,55 @@
+#![warn(rust_2018_idioms)]
```

Comment:
So it looks like this got renamed to https://github.com/tokio-rs/tokio/blob/master/tokio-tls/examples/tls-echo.rs

So I don't think we should add this here.

Reply:
it seems that every file of tokio has this line